src/frontend/src/hooks/use-telescope-contributor.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
+
+const pickRandomContributor = (arrayOfContributors) => {
+  return arrayOfContributors[Math.floor(Math.random() * arrayOfContributors.length) + 1];
+};
+
+const useTelescopeContributor = () => {
+  const [contributor, setContributor] = useState(null);
+  const [error, setError] = useState();
+  const telescopeGitHubContributorsUrl =
+    'https://api.github.com/repos/Seneca-CDOT/telescope/contributors';
+
+  useEffect(() => {
+    const fetchContributors = async () => {
+      try {
+        const response = await fetch(telescopeGitHubContributorsUrl);
+        const arrayOfContributors = await response.json();
```

Comment:
Let's add a bit more error handling here (i.e., we only want to proceed on a 200 response) and also rename that array to drop the `arrayOf` prefix:

```js
if(!response.ok) {
  throw new Error('GitHub API response not OK');
}
const contributors = await response.json();
```